docs/automated_testing.md

```diff
@@ -517,9 +517,11 @@ One special value you can include is `today`. That will insert the date on which
     When I set the variable "court_date" to "today + 12"
 ```
 
+You can also use environment variables to set values with [the `secret variables` Step](#secret-variables-step), even if the value doesn't need to be secret.
+
 ---
 
-The `secret variables` Step can set variables that have sensitive information. For example, a password. The value of this variable will not appear anywhere in the report or in the console. Also, you will be unable to take a screenshot of the page.
+<span id="secret-variables-step">The `secret variables` Step</span> can set variables that have sensitive information. For example, a password. The value of this variable will not appear anywhere in the report or in the console. Also, you will be unable to take a screenshot of the page.
 
 This is a complex Step to use. You **must** use a GitHub "secret" to store the value. To learn how to create and add a secret for the test, see the [GitHub secrets section](#github-secrets).
```

Comment:
This isn't true, since you can use any env var set through the workflow. It's kinda hard to separate the concepts though, given how intertwined everything is. Don't have any suggestions, other than getting rid of the word "must":

```suggestion
This is a complex Step to use. You can use a GitHub "secret" to store the value. To learn how to create and add a secret for the test, see the [GitHub secrets section](#github-secrets).
```